bokeh/server/views/ws.py

```diff
@@ -124,8 +125,10 @@ def on_message(self, fragment):
         raise gen.Return(None)
 
     def on_pong(self, data):
-        #log.debug("received a pong: %r", data)
-        pass
+        try:
+            self.latest_pong = int(codecs.decode(data, 'utf-8'))
+        except:
```

Comment:
At minimum you should catch `Exception`, unless you want to catch `SystemExit` etc. here. However, what exactly can fail here: `int()` or `codecs.decode()` or both? I strongly recommend to catch as specific exception as possible to not hide bugs.